ocrd-import

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 
-function critical { ocrd log -n ocrd-import critical "$1"; }
-function error { ocrd log -n ocrd-import error "$1"; }
-function warning { ocrd log -n ocrd-import warning "$1"; }
-function info { ocrd log -n ocrd-import info "$1"; }
-function debug { ocrd log -n ocrd-import debug "$1"; }
+function critical { echo critical "$1"; }
```

Comment:
If the OCR-D logging facility is too inefficient, I suggest to use a proper mechanism to override it with an additional switch. For example, by aliasing `ocrd log -n ocrd-import` or `echo` to a common `log` backend.

Reply:
Yes, this was just a quick hack to reduce the overhead of calling `ocrd log`.

> For example, by aliasing `ocrd log -n ocrd-import` or `echo` to a common `log` backend.

How do you mean, aliasing?

Reply:
> How do you mean, aliasing?

I meant 
```sh
alias log="ocrd log -n ocrd-import"
...
((fastlog)) && alias log=echo
...
function debug { log debug "$1"; }
...
function critical { log critical "$1"; }
```

Reply:
:tada: that is a nice solution and configurable too.